src/components/Character/styles.scss

```diff
@@ -1,19 +1,134 @@
 .wrapper {
-	font-size: 1em;
+	position: absolute;
+	top: 0;
+	width: 100%;
+	height: 100%;
 
 	&[aria-hidden="true"]{
 		opacity: 0;
 	}
 }
-	.button {
-		&.next {
+
+	.container {
+		position: relative;
+		top: 47vh;
+		padding: 0 var(--padding_container) 0 var(--padding_container);
+		font-size: 1.25rem;
+		--padding_container: 10vw;
+	}
+
+		.name, .nickName, .actorName {
+			line-height: 1;
+		}
+
+		.name {
+			margin-bottom: 0.4em;
+			font-size: 5em;
+			font-family: var(--font_family__heading);
+		}
+
+		.nickName {
+			margin-bottom: 1.35em;
+			font-size: 2.6em;
+			font-family: var(--font_family__heading);
+		}
+
+		.actorName {
+			margin-bottom: 1.7em;
 			font-size: 1em;
+			font-family: var(--font_family__body);
+			text-transform: uppercase;
 		}
-		&.previous {
+
+		.bio {
+			margin-bottom: 2.4em;
 			font-size: 1em;
+			line-height: 1.5;
+			font-family: var(--font_family__body);
+		}
+
+		.hero {
+			position: fixed;
+			top: 0;
+			left: 0;
+			width: 100%;
+			height: 100%;
+			z-index: -1;
+			background-color: crimson;
+		}
+
+		.buttons {
+			font-size: 1.1em;
+			font-family: var(--font_family__heading);
 		}
+
+			.button {
+				display: flex;
+				position: absolute;
+				align-items: center;
+				text-decoration: none;
+				--button_margin: 1.2em;
+			}
+
+			.previous {
+				left: var(--padding_container);
+
+				&:before {
+					display: inline-block;
+					content: "";
+					width: 1.36em;
+					height: 2em;
+					margin-right: var(--button_margin);
+					background-color: lime;
+				}
+			}
+
+			.next {
+				right: var(--padding_container);
+
+				&:after {
+					display: inline-block;
+					content: "";
+					width: 1.36em;
+					height: 2em;
+					margin-left: var(--button_margin);
+					background-color: lime;
+				}
+			}
+
+// Mobile portrait
+
+
+// Mobile landscape
+@media (orientation: landscape) {
+	.container {
+		top: 25vh;
```

Comment:
```suggestion
```
As mentioned above - we just need to center with flexbox here - don't need to push down from the top at all - but if we did, then we should be using `%` here - `vw` and `vh` are for when there's literally no other option. 

They're the weapons that come out, like, 3/4 of the way through the movie when your best friend gets killed and your return back to base, clothes torn, covered in blood; heading to the back of the cave where a slightly out-of-place bookshelf sits.  You shake your head in dismay as you reach out and pull out a book called _Absolutism_; causing the unit to slowly flip over revealing your secret, most powerful stash.

"I can't believe it's come to this", you say as you pick up a `vh` in one hand and a `vw` in the other. "But I have no other option".
"But...but...but wait, mr frank!"
You turn. The ghost of your now deceased amigo, `%`, is standin there, apalled.
"I've got to, little cento." You say, turning away. "There's a container out there whose height needs to be based on the viewport width..."

....

"n..n...no." 😭  little cento whimpers. "what has he become?"
🕶 🌆 🔫

Reply:
😭 poor lil cento!

Reply:
Poor lil cento 😭